sessions/react-effects-and-localstorage/journal-app-localstorage/src/components/EntryForm/index.js

```diff
@@ -0,0 +1,33 @@
+import "./EntryForm.css";
+import Button from "../Button";
+
+function EntryForm({ onAddEntry }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    onAddEntry(data);
+    event.target.reset();
+  }
+
+  return (
+    <form className="entry-form" onSubmit={handleSubmit}>
+      <div className="entry-form__title">New Entry</div>
+      <div className="entry-form__fields">
+        <div className="entry-form__field">
+          <label id="motto">Motto</label>
+          <input type="text" name="motto" htmlFor="motto" />
+        </div>
+        <div className="entry-form__field">
+          <label id="notes">Notes</label>
+          <textarea name="notes" htmlFor="notes" rows="4" />
```

Comment:
I think the `id` and `htmlFor` attributes need to be passed in the exact opposite way:
```suggestion
          <label htmlFor="notes">Notes</label>
          <textarea name="notes" id="notes" rows="4" />
```

Reply:
Good Catch!